cli/rt/file_system.rs

```diff
@@ -78,8 +79,8 @@ impl DenoRtSys {
 
   fn copy_to_real_path(
     &self,
-    oldpath: &Path,
-    newpath: &Path,
+    oldpath: &CheckedPath,
```

Comment:
I'm a bit confused now - I thought that `CheckedPath` is path after normalization/canonicalization, but it turns out that's not true. Can you give some context on the purpose of this struct?

Reply:
It is.

Reply:
I documented the usages of new_unsafe in this file to make this more clear.